README.md

```diff
@@ -771,6 +800,56 @@ transaction = starkinfra.issuingtransaction.get("5155165527080960")
 print(transaction)
 ```
 
+### Issuing Enums
+
+#### Query MerchantCategories
+
+You can query multiple merchant categories according to filters.
+You may use MerchantCategories information to define specific IssuingRule's categories.
+
+```python
+import starkinfra
+
+categories = starkinfra.merchantcategory.query(
+    search="food",
+)
+
+for category in categories:
+    print(category)
+```
+
+#### Query MerchantCountries
+
+You can query multiple merchant countries according to filters.
+You may use MerchantCountries information to define specific IssuingRule's countries.
+
+```python
+import starkinfra
+
+countries = starkinfra.merchantcountry.query(
+    search="brazil",
+)
+
+for country in countries:
+    print(country)
+```
+
+#### Query CardMethods
+
+You can query multiple card methods according to filters.
+You may use CardMethods information to define specific IssuingRule's methods.
```

Comment:
You can query available card methods using this resource.
You may also use CardMethods to define specific purchase method filters in IssuingRules.